themes/default/content/blog/pulumi-release-notes-80/index.md

```diff
@@ -0,0 +1,281 @@
+ 
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+ 
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+   - meagan-cojocar
+ 
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+   - features
+   - pulumi-releases
+ 
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+In addition to our [Cloud Engineering Days launches](/blog/nov-2022-launches), we have been busy shipping improvements in the last 2 months. Let's walk through the release highlights across Pulumi engineering areas from September and October. If you want to learn more between release blogs, follow the CLI improvements in the [pulumi/pulumi repo changelog](https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md) and Pulumi Service features in the [new features blogs](/blog/tag/features).
+
+<!--more-->
+
+- Cloud Engineering Days Launches
+- Cloud Providers and Packages
+  - [New Resources in our Providers](#new-resources-in-our-providers)
+  - [Pulumi+Kubernetes Improvements](#pulumikubernetes-improvements)
+- Pulumi CLI and core technologies
+  - [Pulumi CLI now displays time elapsed per resource](#pulumi-cli-now-displays-time-elapsed-per-resource)
+  - [Automation API NodeJS parallel inline programs](#automation-api-nodejs-parallel-inline-programs)
+  - [Add Output.format to the Python SDK](#add-outputformat-to-the-python-sdk)
+  - [Cloning Pulumi templates from Azure DevOps](#cloning-pulumi-templates-from-azure-devops)
+  - [Skip Checkpoints Experimental Flag](#skip-checkpoints-experimental-flag)
+  - [Token Authentication in Go Providers](#token-authentication-in-go-providers)
+  - [Bump TypeScript version from v3.7.3 to v3.8.3](#bump-typescript-version-from-v373-to-v383)
+  - [Display outputs last in diff view](#display-outputs-last-in-diff-view)
+  - [Support lazy-loading Node modules](#support-lazy-loading-node-modules)
+- Pulumi Service & Pulumi.com
+  - [Pulumi Deployments](#pulumi-deployments)
+  - [Architecture Templates Support](#architecture-templates-support)
+
+## Cloud Providers and Packages
+
+### New resources in our providers
+
+We shipped new versions of the AWS Native provider, Google Native provider and the Azure Native provider that added support for 698 new resources in the last two months. 662 of the new resources were added in the Google Native provider, 17 new resources when excluding IAM policy binding improvements, 29 resources were added to the AWS Native provider and 7 were added to the Azure Native provider.
```

Comment:
This 698 numbers sounds slightly suspiciously large.  If I'm reading right, that because 645 of them were IAM resources?  Probably fine, but maybe worth commenting on why 645 IAM resources showed up?

Reply:
Yeah, I had the same thought - we can mention IAM resources explicitly and link to the PR.